src/AppBundle/Command/ImportTransactionCommand.php

```diff
@@ -0,0 +1,207 @@
+    /** @var OutputInterface */
+    private $output;
+
+    private $locatedExistingLicense = false;
+
+    protected function configure()
+    {
+        $this->setName('app:import:transaction')
+             ->addArgument('file', InputArgument::OPTIONAL, 'Import from file. Overrides remote import')
+             ->addOption("log-json")
+             ->addOption(
+                'new-transaction-notification',
+                null,
+                InputOption::VALUE_NONE,
+                'Sends the email notification when a new transaction is found'
+             );
+    }
+
+    protected function execute(InputInterface $input, OutputInterface $output)
+    {
+        $this->init($input);
+
+        $limit = 50;
+        $offset = 0;
+        $newCnt = 0;
+
+        try {
+            do {
+                if($input->getArgument("file")) {
+                    $json = $this->getLocalFile($input->getArgument("file"));
+                } else {
+                    $json = $this->getTransactions($limit, $offset);
+                }
+                if($input->getOption("log-json")) {
+                    $this->getContainer()->get('logger')->info(var_export($json));
+                }
+//                $total = $json['numSales']; // TODO: find total transactions count in APIv2
+                $newCur = $this->saveTransactions($json['transactions']);
+                $this->em->flush();
+                $newCnt = $newCnt + $newCur;
+                $offset += count($json['transactions']);
+
+                $output->writeln(sprintf('Imported %s of %s transactions, %s new so far', $offset, "TODO", $newCnt));
+            } while ($newCur > 0);
+
+        } catch (\Exception $e) {
+            $output->writeln($e->getMessage());
+
+            return;
+        }
+
+        $output->writeln(sprintf('Imported %s transactions', $offset));
+        $this->getContainer()->get("app.status")->importTransactionDone();
+    }
+
+
+    private function getLocalFile($file)
+    {
+        $content = file_get_contents($file);
+        return json_decode($content,true);
+    }
+
+    private function getTransactions($limit, $offset)
+    {
+        $client = new Client();
+        $response = $client->get(
+            $this->url,
+            [
+                'auth' => [$this->login, $this->password],
+                'query' => ['limit' => $limit, 'offset' => $offset]
+            ]
+        );
+
+        return $response->json();
+    }
+
+    private function saveTransactions($jsonTransactions)
+    {
+        $transactionRepository = $this->em->getRepository('AppBundle:Transaction');
+        $licenseRepository = $this->em->getRepository('AppBundle:License');
+        $addonRepository = $this->em->getRepository('AppBundle:Addon');
+        $newCnt = 0;
+
+        foreach ($jsonTransactions as $jsonTransaction) {
+            if ($transaction = $transactionRepository->findOneBy(["transactionId"=>$jsonTransaction['transactionId']])) {
```

Comment:
@thecatontheflat can you please check your prices for plugins?
as I see, Agile estimate is $125 for 500 users. Since you have $123.75 - probably, thats total for 2 plugins?